spec/support/node_helper.js

```diff
@@ -0,0 +1,20 @@
+__ABLY__ = require('./globals.js.env').__ABLY__;
+__ABLY__.MODULES = require('./modules.js').MODULES;
+
+define = function(requireModules, callback) {
+  var required = requireModules.map(function (moduleName) {
+    var modulePath = (__ABLY__.MODULES[moduleName] || {}).node;
+
+    if (modulePath === 'skip') { return; }
+    if (!modulePath) { console.error("Module " + moduleName + " is not a configured Node module"); }
+
+    return require("../../" + modulePath);
+  });
+
+  callback.apply(this, required);
+}
+
+var reporters = require('jasmine-reporters');
```

Comment:
If you want to use `var`, you will have to wrap this in a function to have a different scope.